tools/shared/src/main/scala/org/scalajs/core/tools/linker/checker/IRChecker.scala

```diff
@@ -1238,7 +1307,8 @@ private final class IRChecker(unit: LinkingUnit,
             reportDuplicateLocalVarName(name)
```

Comment:
I see this crossed my PR for name uniqueness. Do you think we can unify this now? It's a bit awkward to have almost the same loop twice.

Reply:
I will investigate.